github/libp2p.yml

```diff
@@ -4166,6 +4166,9 @@ repositories:
     default_branch: main
     delete_branch_on_merge: true
     description: Use a Reframe service to discover content providers
+    files:
```

Comment:
Unrelated change?

Reply:
It's an automatic fix, because I didn't merge https://github.com/libp2p/github-mgmt/pull/169 before. It'll get rebased in a sec. I'll merge afterwards.